vsintegration/src/vs/FsPkgs/FSharp.VS.FSI/fsiSessionToolWindow.fs

```diff
@@ -431,6 +431,48 @@ type internal FsiToolWindow() as this =
     let showNoActivate() = 
         let frame = this.Frame :?> IVsWindowFrame
         frame.ShowNoActivate() |> ignore
+    
+    let getDebugAttachedFSIProcess () =
```

Comment:
I noticed that you put a space between function name and parens like `f ()`. The existing code is mostly using `f()` without the space. What's the preferred method?